source/includes/templates/tabs-gs.rst

```diff
@@ -16,17 +16,17 @@
    .. raw:: html
 
       <div class="tabs">
-        <ul class="nav nav-tabs nav-justified">
+        <ul tab-strip tab-strip--singleton" role="tablist">
           {{ for tab in tabs getLanguageNames}}
           {{ # Only render the tab here if i < 5 }}
           {{ if i lessThan(5) }}
-          <li{{ if i zero }} class="active"{{ end }}><a href="#{{ tab.id }}">{{ tab.name }}</a></li>
+          <li class="tab-strip__element" data-tabid="{{ tab.id }}" role="tab" aria-selected="{{ if i zero }}true{{ else }}false{{ end }}">{{ tab.name }}</li>
           {{ end }}
           {{ end }}
         </ul>
         <div class="tab-content">
```

Comment:
Below the class was changed to "tabs__content".  I suggest we pick one for consistency. Also below, it has a 

role="tabpanel"